CHANGELOG.md

```diff
@@ -335,14 +377,24 @@ Added
 - Added support for rendering toml states. (#56615)
 - Added `set_path` option for salt-ssh shim. (#56627)
 - Added `win_wua.installed` to check a list of updates that apply to the current Windows build. (#56640)
+- Add gitfs and git_pillar fallback branch (#56647)
 - Added ability to compare package versions in Jinja templates. (#56678)
+- Feature to enable/disable termination protection for ec2 instances (#56703 - Master port of #51096)
+- Add function 'shutdown_host' to vmware cloud (#56711 - Master port of #50177)
+- Add a new drbd monitor function via drbdadm to deprecate the old method of drbd-overview (#56768 - Master port of #50410)
+- Add usergroup support to user.present (#56782 - Master port of #48704)
+- Add ARPCHECK option to rh7_eth template (#56801 - Master port of #49094)
```

Comment:
I'm pretty sure this one is already in here?

Also several `master-port` callouts here